front-end/src/components/layout/Breadcrumbs.js

```diff
@@ -0,0 +1,57 @@
+import { useLocation } from "react-router-dom";
+import MUIBreadcrumbs from "@mui/material/Breadcrumbs";
+import Paper from "@mui/material/Paper";
+import Container from "@mui/system/Container";
+
+import Link from "./Link";
+
+import "./Breadcrumbs.css";
+
+function getPathFriendlyName(path) {
+  const friendlyNames = {
+    register: "Register",
+    login: "Login",
+    find: "Find Tables",
+    "view-table/d+": "View Table",
+    "submit-table": "Submit Table",
+  };
+
+  const entries = Object.entries(friendlyNames);
+  const lookup = entries.map(([regex, name]) => ({ regex, name }));
+  const entry = lookup.find(({ regex }) => path.match(regex));
+
+  return entry?.name;
+}
+
+function createPathBreadcrumbLink(_, index, paths) {
+  const urlPath = paths.slice(0, index + 1).join("/");
+  const friendlyName = getPathFriendlyName(urlPath);
+
+  if (!friendlyName) {
+    return undefined;
+  }
+
+  return (
+    <Link underline="hover" to={urlPath} key={urlPath}>
+      {friendlyName}
+    </Link>
+  );
+}
+
+export default function Breadcrumbs() {
+  const location = useLocation();
+  const paths = location.pathname.split("/").filter((x) => !!x);
+
+  return (
+    <Container maxWidth="md">
+      <Paper className="breadcrumbs-wrapper">

```

Comment:
small thing, but do you think we can make the breadcrumbs a little smaller? they seem to take up a good amount of space, especially on mobile. i guess we'll probably end up messing with the style more once everything is in place, so this is definitely not a big deal.

Reply:
Yeah sounds good - this is the default styling so we can definitely tweak it!

Reply:
Do you want to tweak it in this PR or make another new issue?

Reply:
I'll create a separate issue, I won't have time to take a look at this immediately and if you're OK merging this as-is and having the styling get fixed later, that would be most convenient for me.

Reply:
Tracked in #84